content.go

```diff
@@ -119,13 +118,10 @@ func TagN(ctx context.Context, target Target, srcReference string, dstReferences
 			return err
 		}
 
-		limiter := semaphore.NewWeighted(opts.Concurrency)
-		eg, egCtx := errgroup.WithContext(ctx)
+		eg, egCtx := syncutil.NewLimitGroup(ctx, int(opts.Concurrency))
```

Comment:
There is an issue on casting `int64` to `int`. What if we lose precisions?

@Wwwsylvia Should we change the type of all occurrences of `Concurrency` to `int` from `int64`?

Reply:
Yeah `int` should be enough.

Reply:
Sent a PR for this. See #376